doc/sphinx/GWB_parameter_listings/visualization_grid_file/index.md

```diff
@@ -5,3 +5,32 @@ Visualization Grid file
 ```{todo}
 This part of the manual is used to list all the parameters for the visualization grid file.
 ```
+
+The visualization grid file is used to create a structured mesh and populate the node points with the generated world, i.e., initial conditions of temperature, composition, grains, etc. In essence, it is a text file that contains 3 major sections specifying the mesh points and values :
+
+1. Mesh Output Type
+* grid_type    = annulus|cartesian|chunk|spherical
+The geometry of the output grid. The `gwb-grid` will not throw an error if the chosen grid type is inconsistent with the coordinate system in the input world_builder (`.wb`) file, but the generated output values would not make much sense. This means that for the spherical coordinate system, either of the `annulus`, `spherical`, or `chunk` grid types would give expected output for the computed properties.
+
+* dim          = 2|3
+The dimensions of the world. 
+
+* compositions = 0
+The number of compositions the output .vtu file contains. It can be any number from 0 to MAX_UNSIGNED_INT value. This means that even if we only have 1 composition in the input `.wb` file, it is possible to create visualization outputs for many compositional fields, which will be just equal to zero everywhere in the output mesh. If no value is specified, then it is assumed that there are no compositional fields, i.e., `compositions = 0`.
+
+2. Mesh extent
```

Comment:
I think it would be helpful to mention that even when using spherical geometries you still use x_min, y_min, z_min etc., and also what x, y, and z correspond to in spherical geoemtries.